app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ActiveRecord::Base
+  validates :first_name, :last_name, :password_digest, presence: true
```

Comment:
We don't ever validate `password_digest`, has_secure_password would take care of that for us (or `BCrypt`). 

Since we are going to use auth0, we should follow this [pattern] (https://github.com/nyc-mud-turtles-2015/Soap-Stone/blob/auth-zero/app/models/user.rb)